packager/signer/signer.go

```diff
@@ -179,6 +179,8 @@ func (this *Signer) genCertURL(cert *x509.Certificate, signURL *url.URL) (*url.U
 }
 
 func (this *Signer) ServeHTTP(resp http.ResponseWriter, req *http.Request, params httprouter.Params) {
+	resp.Header().Add("Vary", "Accept")
```

Comment:
While you're here, make it "Accept, AMP-Cache-Transform".

Reply:
I'm slightly partial to a single header `Vary: Accept, AMP-Cache-Transform` as it's slightly simpler (intermediaries are supposed to handle duplicate header names, but I wouldn't be surprised if some don't).